include/indexstore/indexstore.h

```diff
@@ -84,6 +84,28 @@
 # define INDEXSTORE_NOESCAPE
 #endif
 
+#if __has_attribute(flag_enum)
+# define _INDEXSTORE_FLAG_ENUM __attribute__((flag_enum))
```

Comment:
`_[A-Z]` is a reserved identifier in C.  How about `INDEXSTORE_FLAG_ENUM_ATTR`, `INDEXSTORE_OPEN_ENUM_ATTR` and `INDEXSTORE_OPTIONS_ATTRS`?

Reply:
Does this apply to for macros too? I was following suit with `CFAvailability.h` (ex `__CF_ENUM_ATTRIBUTES`).

Reply:
Yes it does.  `CFAvailability.h` is effectively part of the system (at least on Apple platforms), so it gets to use reserved names.  Same for e.g. the libc or libc++.  While this header is in the compiler repo, it's probably best to treat it more like a normal library header and avoid reserved identifiers.

Reply:
👍 Good to know.